docker-compose.yml

```diff
@@ -17,6 +17,14 @@ services:
       context: backend
       dockerfile: Dockerfile.dev
     ports:
-      - 5000:5000
+      - 8000:8000
     volumes:
       - ./backend:/app
+  db:
```

Comment:
Mismatch of `db` key in docker-compose and `postgres` host name from the django setting